### ISSUE TITLE
Many unknown source names in CmpStatus out of the box

### ISSUE THREAD

**Issue by chao-mu** [LunarVim/LunarVim#4435]
### Problem description

Executing `:CmpStatus` shows this:

```
# ready source names
- luasnip
- buffer
- path

# unknown source names
- copilot
- nvim_lsp
- cmp_tabnine
- nvim_lua
- calc
- emoji
- treesitter
- crates
- tmux
```

Config file: I emptied it to see if this still was an issue, it was.

### LunarVim version

 release-1.3/neovim-0.9-d1c1bace

### Neovim version (>= 0.9.1)

NVIM v0.9.4

### Terminal name

GNOME Terminal

### Operating system/version

Ubuntu 23.04

### Steps to reproduce

1. lvim
2. :CmpStatus

### Healthcheck

No errors. The only warnings are:

(wish I knew how to update mason itself)
```
mason.nvim ~
- WARNING mason.nvim version v1.0.1
  - ADVICE:
    - The latest version of mason.nvim is: v1.8.3
```

```
mason.nvim [Languages] ~
- WARNING luarocks: not available
  - ADVICE:
    - spawn: luarocks failed with exit code - and signal -. luarocks is not executable
- WARNING Composer: not available
  - ADVICE:
    - spawn: composer failed with exit code - and signal -. composer is not executable
- WARNING PHP: not available
  - ADVICE:
    - spawn: php failed with exit code - and signal -. php is not executable
- WARNING javac: not available
  - ADVICE:
    - spawn: javac failed with exit code - and signal -. javac is not executable
- WARNING julia: not available
  - ADVICE:
    - spawn: julia failed with exit code - and signal -. julia is not executable
```

(I've tried creating this file, but the warning doesn't go away)
```
Configuration ~
- WARNING Missing user config file: /home/danimal/.config/lvim/init.vim
  - ADVICE:
    - :help |init.vim|
```

**Comment by chao-mu** [comment]
I am VERY confused why copilot is showing up in my missing sources as I don't see it here: https://github.com/abzcoding/lvim/blob/main/lua/user/builtin.lua

**Comment by chao-mu** [comment]
Wanted to make sure I did a truly clean install, so I used updatedb/locate to find every instance of lvim and lunarvim I could find and removed them.

Yet still

```
# ready source names
- luasnip
- buffer
- path

# unknown source names
- copilot
- nvim_lsp
- cmp_tabnine
- nvim_lua
- calc
- emoji
- treesitter
- crates
- tmux
```

**Comment by chao-mu** [comment]
Used the official lvim uninstaller and did a clean install, same issue.

**Comment by chao-mu** [comment]
Here's the result of `:Lazy`

 ```
  Total: 43 plugins
  
  Loaded (12)
    ● alpha-nvim 0.36ms  VimEnter
    ● cmp-buffer 1.05ms   nvim-cmp
    ● cmp-nvim-lsp 1.17ms   nvim-cmp
    ● cmp-path 0.79ms   nvim-cmp
    ● cmp_luasnip 4.22ms   nvim-cmp
    ● lazy.nvim 1.87ms   init.lua
    ● lualine.nvim 3.21ms  VimEnter
    ● lunar.nvim 0.02ms  start
    ● nvim-cmp 8.9ms  CmdlineEnter
    ● nvim-web-devicons 0.08ms   lualine.nvim
    ● project.nvim 0.42ms  VimEnter
    ● which-key.nvim 5.27ms  VeryLazy
  
  Not Loaded (30)
    ○ Comment.nvim  FileOpened   gc gb
    ○ LuaSnip  InsertEnter
    ○ bigfile.nvim  FileOpened
    ○ bufferline.nvim  FileOpened
    ○ friendly-snippets   LuaSnip
    ○ gitsigns.nvim   Gitsigns  FileOpened
    ○ indent-blankline.nvim  FileOpened
    ○ lir.nvim  DirOpened
    ○ mason-lspconfig.nvim   LspInstall LspUninstall  FileOpened   nvim-lspconfig
    ○ mason.nvim   Mason MasonInstall MasonUninstall MasonUninstallAll MasonLog  FileOpened   mason-lspconfig.nvim
    ○ neodev.nvim 
    ○ nlsp-settings.nvim   LspSettings   nvim-lspconfig
    ○ null-ls.nvim 
    ○ nvim-autopairs  InsertEnter
    ○ nvim-dap 
    ○ nvim-dap-ui   nvim-dap
    ○ nvim-lspconfig 
    ○ nvim-navic  FileOpened
    ○ nvim-tree.lua   NvimTreeToggle NvimTreeOpen NvimTreeFocus NvimTreeFindFileToggle  DirOpened
    ○ nvim-treesitter   TSInstall TSUninstall TSUpdate TSUpdateSync TSInstallInfo TSInstallSync TSInstallFromGrammar  FileOpened   nvim-autopairs
    ○ nvim-ts-context-commentstring 
    ○ onedarker.nvim 
    ○ plenary.nvim   PlenaryBustedFile PlenaryBustedDirectory
    ○ schemastore.nvim 
    ○ structlog.nvim 
    ○ telescope-fzf-native.nvim   telescope.nvim
    ○ telescope.nvim   Telescope
    ○ toggleterm.nvim   ToggleTerm TermExec ToggleTermToggleAll ToggleTermSendCurrentLine ToggleTermSendVisualLines ToggleTermSendVisualSelection   <c-\>
    ○ tokyonight.nvim 
    ○ vim-illuminate  FileOpened
  
  Disabled (1)
    ○ cmp-cmdline   nvim-cmp
```

**Comment by chao-mu** [comment]
Did a clean install of nvim. Same issue.

**Comment by chao-mu** [comment]
I fixed it after copying a random config off the internet and running `:Lazy sync`. Thinking back, I am not sure I tried running the latter on some of the fresh installs. Usually what happens when I do a fresh install is that there's an error and it tells me to Lazy sync, but this time there were no such errors, so I didn't think I needed to run it, I guess. Or maybe that had nothing to do with what ultimately solved the issue.

Well, at least I learned a TON about nvim and lunarvim.
